src/utils.py

```diff
@@ -22,3 +24,26 @@ def valid_date(s):
     except ValueError:
         msg = "not a valid date: {0!r}".format(s)
         raise argparse.ArgumentTypeError(msg)
+
+
+class Network(Enum):
+    MAINNET = "mainnet"
+    GNOSIS = "gnosis"
+
+    def dune_v1_repr(self) -> LegacyDuneNetwork:
+        return {
+            Network.MAINNET: LegacyDuneNetwork.MAINNET,
+            Network.GNOSIS: LegacyDuneNetwork.GCHAIN,
+        }[self]
+
+    def dune_v2_repr(self) -> str:
+        return {Network.MAINNET: "ethereum", Network.GNOSIS: "gnosis"}[self]
+
+    def node_url(self) -> str:
```

Comment:
Nitpick: Missing verb here. What do you think of `get_node_url`?

Reply:
I am not really a fan of putting GET everywhere (since it generally reminds me of a request as if we were fetching something)

Perhaps I should tag this function with a @property decorator

Because these are attributes of a Network instance


Network.chain_id (when network is ethereum) read as ethereum chain id. Why do we need to "get" it?
Network.node_url

anyway. I am not in favour of these suggestions. I hope its alright with you. Generally I think "getters" are actually expected to "get" something from somewhere else.

Reply:
I think the property decorator is a very good suggestion for this. 

For the sake of the discussion: get and set methods work with variables inside of classes and get and set those variables without exposing them, at least for OOP languages. Although, I also agree that in python's case that this doesn't make as much sense and can impair readability.